lib/v2/liveuamap/radar.js

```diff
@@ -0,0 +1,13 @@
+module.exports = {
+    liveuamap: {
+        _name: 'Live Universal Awareness Map',
+        '.': [
+            {
+                title: 'Region',
+                docs: 'https://docs.rsshub.app/routes/new-media#liveuamap',
+                source: ['/:region*'],
+                target: '/:region',
```

Comment:
https://docs.rsshub.app/joinus/new-radar#target
```suggestion
                target: '/liveuamap/:region',
```